sirepo/job_api.py

```diff
@@ -74,6 +74,20 @@ async def api_beginSession(self):
     )
     async def api_downloadDataFile(
         self, simulation_type, simulation_id, model, frame, suffix=None
+    ):
+        return await self.api_downloadRunFile(
```

Comment:
Mark this as deprecated. It's a special case to call an api directly like this so we want to document that this shouldn't be done in general. See auth.github.api_oauthAuthorized as an example.

Reply:
[f5c59d4](https://github.com/radiasoft/sirepo/pull/6348/commits/f5c59d418a77512127ab3a6c851cec6a13f139f5)